### ISSUE TITLE
Ubuntu 20.04 [ getLoginName: does not exist (No such device or address) ] when program is ran

### ISSUE THREAD

**Issue by chinatimmy** [kadena-io/chainweaver#636]
Using Xubuntu 20.04.1 LTS x86_64 

**ran command** 
redacted@redacted:/usr/libexec/chainweaver$ ./kadena-chainweaver 

**output produced**
Executable path: /usr/libexec/chainweaver/
kadena-chainweaver: getLoginName: does not exist (No such device or address)

**recommendation/enhancement**
App image or Flatpak as some development teams don't all use the same distro

**Comment by jmininger** [comment]
Spent a bit of time looking into this...The following thread seems relevant, along with the linked SO q/a: 
https://gitlab.haskell.org/ghc/ghc/-/issues/1487
https://stackoverflow.com/questions/4785126/getlogin-c-function-returns-null-and-error-no-such-file-or-directory

with the chainweaver code here, being the chunk in question: https://github.com/kadena-io/chainweaver/blob/develop/linux/src/Linux.hs#L178

**Comment by freQniK** [comment]
Yeah, I'm getting the same thing on recent build still. Ubuntu 20.04. Not a very good wallet if it doesn't even load.

**Comment by trendzetter** [comment]
https://discord.com/channels/502858632178958377/631965054778212362/948882072490098708

**Comment by vladimir-litvinenko** [comment]
@trendzetter @freQniK  Are there any progress on merging this? Just installed .deb on Ubuntu. I'm experiencing the same issue:

```
 > sudo dpkg -i kadena-chainweaver-linux-2.2.2.0.deb 
(Reading database ... 704403 files and directories currently installed.)
Preparing to unpack kadena-chainweaver-linux-2.2.2.0.deb ...
Unpacking kadena-chainweaver (2.2.2.0) over (2.2.2.0) ...
Setting up kadena-chainweaver (2.2.2.0) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.24+linuxmint1) ...
Processing triggers for mime-support (3.64ubuntu1) ...

 > kadena-chainweaver 
Executable path: /usr/libexec/chainweaver/
kadena-chainweaver: getLoginName: does not exist (No such device or address)
 
```
